frontend/src/fixtures/commonsFixtures.js

```diff
@@ -35,7 +35,66 @@ const commonsFixtures = {
                 "totalPlayers": 50,
                 "cowPrice": 15,
             }
-        ]
+        ],
+
+    sevenCommons: [
+        {
+            "id": 10,
+            "name": "Seths Commons",
+            "day": 5,
+            "endDate": "6/11/2021",
```

Comment:
These fixtures are dated, but that's okay. We're not rendering them. Yet. Should be an ISO string.